Resolve g++ warnings about implicit conversion

```
Graphics.cpp:33:11: warning: implicit conversion of NULL constant to 'int'
      [-Wnull-conversion]
        mWidth = NULL;
               ~ ^~~~
                 0
Graphics.cpp:34:12: warning: implicit conversion of NULL constant to 'int'
      [-Wnull-conversion]
        mHeight = NULL;
                ~ ^~~~
                  0
Graphics.cpp:470:73: warning: implicit conversion of NULL constant to 'Uint32'
      (aka 'unsigned int') [-Wnull-conversion]
  ...= SDL_ConvertSurface(loadedSurface, surface->format, NULL);
       ~~~~~~~~~~~~~~~~~~                                 ^~~~
                                                          0
3 warnings generated.
```

--- a/src/Graphics.cpp
+++ b/src/Graphics.cpp
@@ -14,8 +14,8 @@ LTexture* Graphics::getMyForeground()
 LTexture::LTexture()
 {
 	mTexture = NULL;
-	mWidth = NULL;
-	mHeight = NULL;
+	mWidth = 0;
+	mHeight = 0;
 }
 LTexture::~LTexture()
 {
@@ -451,8 +451,8 @@ SDL_Surface* Graphics::loadSurface(string filename)
 	else
 	{
 		SDL_Surface* surface = SDL_GetWindowSurface(window);
-		optimizedSurface = SDL_ConvertSurface(loadedSurface, surface->format, NULL);
-		
+		optimizedSurface = SDL_ConvertSurface(loadedSurface, surface->format, 0);
+
 		if (optimizedSurface == NULL)
 		{
 			cerr << "Surface could not be converted" << endl;

